bundles/org.openhab.binding.easee/src/main/java/org/openhab/binding/easee/internal/model/ChargerOpState.java

```diff
@@ -0,0 +1,49 @@
+package org.openhab.binding.easee.internal.model;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+@NonNullByDefault
```

Comment:
Please add file headers and javadoc including @author tag

Reply:
I always forget this :-(